src/store.ts

```diff
@@ -124,6 +149,15 @@ const store: TypedVuexStore = new TypedVuexStore({
     ) {
       state.selectableRequirementChoices = selectableRequirementChoices;
     },
+    setOverridenRequirementChoices(
+      state: VuexStoreState,
+      overridenRequirementChoices: AppOverridenRequirementChoices
+    ) {
+      state.overridenRequirementChoices = {
+        ...overridenRequirementChoices,
+        ...computeAPIBOverridenRequirements(state),
```

Comment:
Can you add a comment about how this adds overridden requirements from AP/IB data on Firestore?